pkg/dbmate/db_test.go

```diff
@@ -315,17 +316,13 @@ func TestWaitBeforeVerbose(t *testing.T) {
 	output := capturer.CaptureOutput(func() {
 		testWaitBefore(t, true)
 	})
-	require.Contains(t, output,
-		`Applying: 20151129054053_test_migration.sql
-Last insert ID: 1
-Rows affected: 1
-Applying: 20200227231541_test_posts.sql
-Last insert ID: 1
-Rows affected: 1`)
-	require.Contains(t, output,
-		`Rolling back: 20200227231541_test_posts.sql
-Last insert ID: 0
-Rows affected: 0`)
+	matched, err := regexp.MatchString(`Applying: 20151129054053_test_migration\.sql\nLast insert ID: 1\nRows affected: 1\nApplied: 20151129054053_test_migration\.sql in ([\w.,µ]+)\nApplying: 20200227231541_test_posts\.sql\nLast insert ID: 1\nRows affected: 1\nApplied: 20200227231541_test_posts\.sql in ([\w.,µ]+)`, output)
+	require.NoError(t, err)
+	require.True(t, matched)
+
+	matched, err = regexp.MatchString(`Rolling back: 20200227231541_test_posts\.sql\nLast insert ID: 0\nRows affected: 0\nRolled back: 20200227231541_test_posts\.sql in ([\w.,µ]+)`, output)
+	require.NoError(t, err)
+	require.True(t, matched)
```

Comment:
This doesn't yield a particularly useful diagnostic message on failure:

```
=== RUN   TestWaitBeforeVerbose
    db_test.go:321: 
                Error Trace:    /src/pkg/dbmate/db_test.go:321
                Error:          Should be true
                Test:           TestWaitBeforeVerbose
--- FAIL: TestWaitBeforeVerbose (0.14s)
```

How do you feel about this as an alternative?

```
func TestWaitBeforeVerbose(t *testing.T) {
	output := capturer.CaptureOutput(func() {
		testWaitBefore(t, true)
	})

	re := regexp.MustCompile(`((Applied|Rolled back): .* in) ([\w.,µ]+)`)
	maskedOutput := re.ReplaceAllString(output, "$1 ELAPSED")

	require.Contains(t, maskedOutput,
		`Applying: 20151129054053_test_migration.sql
Last insert ID: 1
Rows affected: 1
Applied: 20151129054053_test_migration.sql in ELAPSED
Applying: 20200227231541_test_posts.sql
`)
	require.Contains(t, maskedOutput,
		`Rolling back: 20200227231541_test_posts.sql
Last insert ID: 0
Rows affected: 0
Rolled back: 20200227231541_test_posts.sql in ELAPSED`)
}
```

Granted, the test output isn't easy to read, but at least you have the full expected and actual strings to be able to deduce what was mismatched that caused the test to fail:

```
=== RUN   TestWaitBeforeVerbose
    db_test.go:330: 
                Error Trace:    /src/pkg/dbmate/db_test.go:330
                Error:          "Dropping: dbmate_test.sqlite3\nWaiting for database.....\nCreating: dbmate_test.sqlite3\nWaiting for database.....\nApplying: 20151129054053_test_migration.sql\nLast insert ID: 1\nRows affected: 1\nApplying: 20200227231541_test_posts.sql\nLast insert ID: 1\nRows affected: 1\nWaiting for database.....\nWaiting for database.....\nRolling back: 20200227231541_test_posts.sql\nLast insert ID: 0\nRows affected: 0\nRolled back: 20200227231541_test_posts.sql in ELAPSED\nWaiting for database.....\nWriting: ./db/schema.sql\nWaiting for database.....\nDropping: dbmate_test.sqlite3\nCreating: dbmate_test.sqlite3\nReading: ./db/schema.sql\nLast insert ID: 1\nRows affected: 1\nWaiting for database.....\n" does not contain "Applying: 20151129054053_test_migration.sql\nLast insert ID: 1\nRows affected: 1\nApplied: 20151129054053_test_migration.sql in ELAPSED\nApplying: 20200227231541_test_posts.sql\n"
                Test:           TestWaitBeforeVerbose
--- FAIL: TestWaitBeforeVerbose (0.15s)
```

Reply:
@dossy thanks for your comment; I've made changes. Please check them over.